packages/lib/models/Setting.ts

```diff
@@ -1384,7 +1384,28 @@ class Setting extends BaseModel {
 				label: () => _('Ignore TLS certificate errors'),
 				storage: SettingStorage.File,
 			},
-
+			'sync.proxyUrl': {
+				value: '',
+				type: SettingItemType.String,
+				advanced: true,
+				section: 'sync',
+				isGlobal: true,
+				public: true,
+				label: () => _('Proxy URL'),
+				description: () => _('This can be set here or via http_proxy/https_proxy env var'),
+				storage: SettingStorage.File,
+			},
+			'sync.proxyTimeout': {
+				value: 1,
+				type: SettingItemType.Int,
+				advanced: true,
+				section: 'sync',
+				isGlobal: true,
+				public: true,
+				label: () => _('Proxy Timeout (Seconds)'),
+				description: () => _('(Seconds) How long should be spent making a connection to the proxy'),
```

Comment:
Please remove the description (it's one more thing to translate and it's not so useful)